Solutions/MaturityModelForEventLogManagementM2131/Workbooks/MaturityModelForEventLogManagement_M2131.json

```diff
@@ -1174,7 +1209,7 @@
           {
             "type": 1,
             "content": {
-              "json": "### Pricing\r\n✳️[Microsoft Sentinel: Ingestion](https://azure.microsoft.com/pricing/details/azure-sentinel/#pricing)<br>\r\n✳️[Log Analytics: Log Retention](https://azure.microsoft.com/pricing/details/monitor/#pricing)"
+              "json": "### Pricing\r\n✳️ [Microsoft Sentinel: Ingestion](https://azure.microsoft.com/pricing/details/azure-sentinel/#pricing)<br>\r\n✳️ [Log Analytics: Log Retention](https://azure.microsoft.com/pricing/details/monitor/#pricing)"
```

Comment:
I noticed you removed some of the icons in other parts, did you want to remove here also or not?

Reply:
Chatted with TJ - this is by design.  Closing ocmment.